Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1216,6 +1220,20 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C11M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal'],
     },
+    '2026D78' : {
+        'Geom' : 'Extended2026D78',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T22',
```

Comment:
@srimanob this is configured incorrectly.
Please add:
```python
 'ProcessModifier': 'PixelCPEGeneric',   # This swaps template reco CPE for generic reco CPE`
 ```
 as done here:
 
 https://github.com/cms-sw/cmssw/blob/db968ce4dce2f0fd9928d40c209fc2bbbf1ac3f7/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1130 
 
 for geometry D64.

Reply:
Fixed, Thanks.